sdk/core/src/request_options/accept.rs

```diff
@@ -0,0 +1,29 @@
+use crate::headers::{self, Header};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct Accept<'a>(&'a str);
```

Comment:
Can we use `String` or `Cow<'static, str>` instead of `&'a str`?

Reply:
Used `String` since this seems more "common" right now. Was almost about to migrate more headers, but that's better left for another PR :)